.mergify.yml

```diff
@@ -117,21 +117,21 @@ pull_request_rules:
         labels:
           - feature-gate
         branches:
-          - v2.2
-  - name: v2.2 non-feature-gate backport
+          - v3.0
+  - name: v3.0 non-feature-gate backport
     conditions:
-      - label=v2.2
+      - label=v3.0
       - label!=feature-gate
     actions:
       backport:
         assignees: *BackportAssignee
         title: "{{ destination_branch }}: {{ title }} (backport of #{{ number }})"
         ignore_conflicts: true
         branches:
-          - v2.2
-  - name: v2.2 backport warning comment
+          - v3.0
+  - name: v3.0 backport warning comment
     conditions:
-      - label=v2.2
+      - label=v3.0
```

Comment:
I think this should be updated to v2.3, and the block below to v3.0.

The comment below this says "stable" branch which is v2.3; the message even lower in the current v2.3 block says "beta branch" (which is what v3.0 is)

Reply:
We can have variables AFAIK. Lemme take a look and see how that would work.

Reply:
Also, I'm fine without variables for this PR; but we do need to make sure the tags line up with stable / beta correctly; we can fix that manually if we just want to make sure that everything is in place and cleanup later

Reply:
Sadly, the mergify schema does not allow variables.
I've updated the PR with the correct tags.